src/dpq2/conv/to_bson.d

```diff
@@ -141,6 +141,11 @@ Bson rawValueToBson(in Value v)
             auto b = BsonBinData(BsonBinData.Type.userDefined, v.data.idup);
             res = Bson(b);
             break;
+        case VariableBitString:
+            import std.format;
+            auto dval = v.tunnelForBinaryValueAsCalls!BitArray;
+            res = Bson(format("%b", dval));
```

Comment:
(For datetime type I made an exception in support by Bson simply because datetime is a difficult thing in PG and I did not want to leave for users opportunity to make a stupid mistake. But with varbit users will figure it out without problems themselves, I think.)